.github/workflows/build.yml

```diff
@@ -41,4 +41,4 @@ jobs:
         
       - name: Deploy with mike
         run: |
-          mike deploy --push $(cat version.txt)
\ No newline at end of file
+          mike deploy --push $(cat version.txt) latest
```

Comment:
We may need update-aliases here as well
` mike deploy --push --update-aliases $(cat version.txt) latest`